addons/google_calendar/tests/test_sync_odoo2google.py

```diff
@@ -798,3 +808,19 @@ def test_allday_duplicated_first_event_in_recurrence(self):
             'extendedProperties': {'shared': {'%s_odoo_id' % self.env.cr.dbname: recurrence.id}},
             'transparency': 'opaque',
         }, timeout=3)
+
+    @patch_api
+    def test_partner_type_change(self):
+        user = self.user1
+        event = self.env['calendar.event'].create({
+            'name': "Private Event",
+            'user_id': user.id,
+            'start': datetime(2020, 1, 13, 16, 55),
+            'stop': datetime(2020, 1, 13, 19, 55),
+            'partner_ids': [(4, self.private_partner.id)],
```

Comment:
I would've preferred teh Command syntax, but I see it's not used in existing tests, it's not even imported, so let's keep it as it is.

Reply:
Me too.